src/Parser/Shaman/Restoration/CombatLogParser.js

```diff
@@ -145,24 +146,29 @@ class CombatLogParser extends CoreCombatLogParser {
     const unbuffedHealingSurges = healingSurges - twHealingSurges;
     const totalTwUsed = twHealingWaves + twHealingSurges;
     const giftOfTheQueenCasts = giftOfTheQueen.casts || 0;
+    const giftOfTheQueenDuplicateCasts = giftOfTheQueen.casts || 0;
 
     const chainHealHits = chainHeal.healingHits || 0;
     const chainHealAvgHits = chainHealHits / chainHealCasts;
     const maxChainHealTargets = this.modules.combatants.selected.hasTalent(SPELLS.HIGH_TIDE_TALENT.id) ? 5 : 4;
     const chainHealTargetEfficiency = chainHealAvgHits / maxChainHealTargets;
 
-    const hasDeepWaters = this.modules.combatants.selected.traitsBySpellId[SPELLS.DEEP_WATERS.id] > 0;
     const giftOfTheQueenHits = giftOfTheQueen.healingHits || 0;
-    const giftOfTheQueenAvgHits = giftOfTheQueenHits / giftOfTheQueenCasts / (hasDeepWaters ? 2 : 1);
+    const giftOfTheQueenDuplicateHits = giftOfTheQueenDuplicate.healingHits || 0;
+    const giftOfTheQueenAvgHits = (giftOfTheQueenHits+giftOfTheQueenDuplicateHits) / (giftOfTheQueenCasts+giftOfTheQueenDuplicateCasts);
     const giftOfTheQueenTargetEfficiency = giftOfTheQueenAvgHits / 6;
```

Comment:
Question: Does the target efficiency change with the additional of the Deep Waters trait?

I would believe this would be modified to be either 6 or 12 targets, depending of if you have the trait or not?

Also, if that is the case, does the lower suggestion change?

Reply:
It doesn't need to adjust to 12 target if you divide by the sum of Normal and Duplicate casts.
You could also just take the amount of normal casts and divide by 12, but you would have a weird edge case if the fight ends/you die in the 3 second window before the second cast goes through

Reply:
Makes sense. Understood.